src/services/translator/Export_ImportTranslationService.php

```diff
@@ -109,7 +109,7 @@ public function updateDraft($element, $draft, $translatedContent, $sourceSite, $
                 $draft->title = isset($targetData['title']) ? $targetData['title'] : $draft->title;
                 $draft->slug = isset($targetData['slug']) ? $targetData['slug'] : $draft->slug;
 
-				$post = Translations::$plugin->elementTranslator->toPostArrayFromTranslationTarget($draft, $sourceSite, $targetSite, $targetData);
+				$post = Translations::$plugin->elementTranslator->toPostArrayFromTranslationTarget($element, $sourceSite, $targetSite, $targetData);
```

Comment:
@shnsumit @bhupeshappfoster could you please explain the change from `$draft` to `$element`? This may have unintended consequences.

Reply:
@sidedwards 
this change is because of drafts in target sites never contains localised nested blocks and hence can lead missing blocks from draft so we use source entry to get all content to target in case it is missing from target

Reply:
@bhupeshappfoster, thanks for the explanation. We will need to do some comprehensive e2e testing to make sure this doesn't break other workflows.

Reply:
this change from draft to element is not required for the fix give me min i ll commit a revert and then we can proceed with merge and release

Reply:
@sidedwards 
Please review again i think we can proceed with merge now.